runtime/tools/python/ttrt/common/run.py

```diff
@@ -450,6 +457,7 @@ def _execute(binaries):
                                     program_index,
                                     total_inputs[loop],
                                     total_outputs[loop],
+                                    self["--use-graph-capture"],
```

Comment:
can we add this in trt.runtime.DebugEnv? It seems like this feature is a debug mode feature and the infra is already there to add specific bools like this one

debug_env = ttrt.runtime.DebugEnv.get(
                self["--load-kernels-from-disk"], self["--enable-async-ttnn"]
            )